assets/Broadcast_page/README.md

```diff
@@ -0,0 +1 @@
+# BroadCast Page created by JLopezz3
```

Comment:
can you please add the figma link in here?

Reply:
Okay

Reply:
Done. I added the Figma link in the README